src/Microsoft.ML.Transforms/OneHotEncoding.cs

```diff
@@ -22,7 +22,32 @@
 
 namespace Microsoft.ML.Transforms
 {
-    /// <include file='doc.xml' path='doc/members/member[@name="CategoricalOneHotVectorizer"]/*' />
+    /// <summary>
+    /// <see cref="ITransformer"/> resulting from fitting a <see cref="OneHotEncodingEstimator"/>.
+    /// </summary>
+    /// <remarks>
+    /// <para>
```

Comment:
please use the template.

remarks are in markdown and include io table. please see similar merged PRs.